src/samplers.jl

```diff
@@ -10,7 +10,8 @@ for fname in ["aliastable.jl",
               "vonmises.jl",
               "vonmisesfisher.jl",
               "discretenonparametric.jl",
-              "categorical.jl"]
+              "categorical.jl",
+              "resampler.jl"]
```

Comment:
```suggestion
              "resampler.jl",
]
```

Like this it will not modify two lines to add a new file